README.md

```diff
@@ -35,6 +36,9 @@ To build the code without running the tests, add to the "clean build" command th
 > ./gradlew clean build -x test
 ```
 
+### Running pip tests
+Pip tests must run inside a clean pip-environment. Create a virtual environment and provide the path to its '/bin' directory using the 'BITESTS_ARTIFACTORY_PIP_ENV' variable.
```

Comment:
I suggest adding clear step-by-steps instructions here.
We should consider restructuring the instructions - having this section and the next one separately, make it hard to understand what should be done to run the tests.

Reply:
We are going to change the README structure to be more usable.
Merging the version proposed in this PR and will work on improving it in a new one.